demos/node-classification/cluster-gcn/cluster-gcn-node-classification.ipynb

```diff
@@ -357,13 +357,13 @@
     {
```

Comment:
<p>Missing a plot here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
ah.. dunno how that happened but nice catch!